### DISCUSSION TITLE
4.3 4.4版本启动报错 Error:default level:5

### DISCUSSION THREAD

**Discussion by renqirong** [qier222/YesPlayMusic#1319] [Q&A] [answered]
![QQ截图20220219011706](https://user-images.githubusercontent.com/57310595/154730890-3d937e5c-ac2a-4c20-9aa6-2078e6514a08.png)

**Comment by pan93412** [comment, marked as answer]
進到環境變數設定，然後試試看把 `LOG_LEVEL` 的值刪掉。應該是這個值跟 YPM 衝突了。
